packages/client/src/src-sw.js

```diff
@@ -81,9 +89,31 @@ workbox.routing.registerRoute(
   new workbox.strategies.NetworkFirst()
 )
 
+// This caches the minio urls
+workbox.routing.registerRoute(
+  import.meta.env.PROD
+    ? /http(.+)document\.opencrvs\.org\/+/
+    : /http(.+)localhost:9000\/+/,
```

Comment:
I think we somehow need to use the `MINIO_URL` constant for this regex because it's not guaranteed that the URL pattern will be like this for every country implementation